### PR TITLE
[WIP] Add the document about automation

### PR DESCRIPTION
**Pull request by CharlleDaniel**
This pr is able to:
- Add the documentation about automation

### CODE REVIEW

File: physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+=== Creating, Configuring &nbsp;and Assigning a new Policy
```
Comment (skovic) [inline physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc]: Did you mean to leave this section empty?

File: physical_infra_provider_configuration/automation/topics/copy_lenovo_xclarity_class.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+==== Copy the &nbsp;LenovoXclarity class to the domain created
```
Comment (skovic) [inline physical_infra_provider_configuration/automation/topics/copy_lenovo_xclarity_class.adoc]: It just seems to be adding an extra space, but you may want to remove _&nbsp_ .

File: physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
+==== Creating a new instance to the LenovoXclarity class
+
+1.  Select your domain created and drill down Your Domain name→ &nbsp;System → Event → EmsEvent →LenovoXclarity and select **(LenovoXclarity)**.
```
Comment (skovic) [inline physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: It just seems to be adding an extra space, but you may want to remove _&nbsp_ .

File: physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc (hunk 1 above)
Comment (skovic) [reply physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc]: Also, if you keep this section, you may want to remove _&nbsp_ .

File: physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+3.  Enter with the name
+
+    a.  The name must be equal to event name that will be managed, because the Cloud Forms match the instance with the event by the name.
+    
+    b.  Find the name of the event that you wish in this link **<TO DO>**
+    
+    c.  Copy the name of the event and past in the field’s name in Cloud Forms **(ex. FQXHMTS0003G)**.
+
+4.  Add the path to the handlers copied in the field’s **rel3**.
```
Comment (skovic) [inline physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: What is rel3?

Comment (CharlleDaniel) [reply physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: Hi Stephen, sorry about the "&nbsp", I think that it is a problem in my text editor. Firstly thank you for your review and the rel3 is the name of the field (Relationship three). If you have any suggestions about that, please let me know.

File: physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc [hunk 5]
```diff
@@ -0,0 +1 @@
+=== Creating, Configuring and Assigning a new Policy
```
Comment (CharlleDaniel) [inline physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc]: @skovic. I've created this empty sub section for better organize, like this example:
image

File: physical_infra_provider_configuration/automation/topics/copy_lenovo_xclarity_class.adoc (hunk 2 above)
Comment (CharlleDaniel) [reply physical_infra_provider_configuration/automation/topics/copy_lenovo_xclarity_class.adoc]: done

File: physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc (hunk 1 above)
Comment (CharlleDaniel) [reply physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc]: done

File: physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc (hunk 5 above)
Comment (skovic) [reply physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc]: OK, got it, thanks

File: physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc (hunk 4 above)
Comment (skovic) [reply physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: @CharlleDaniel So _rel3_ is the name of a field in the ManageIQ UI?

File: physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+1.  Select your domain created and drill down Your Domain name→ System → Event → EmsEvent →LenovoXclarity and select **(LenovoXclarity)**.
+
+2.  Click **(Configuration)**, then click **(Add a new Instance)**.
+
+3.  Enter with the name
+
+    a.  The name must be equal to event name that will be managed, because the Cloud Forms match the instance with the event by the name.
+    
+    b.  Find the name of the event that you wish in this link **<TO DO>**
```
Comment (skovic) [inline physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: @CharlleDaniel Did you intend to leave this TO DO placeholder?

File: physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc (hunk 4 above)
Comment (CharlleDaniel) [reply physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: yes

File: physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc (hunk 6 above)
Comment (CharlleDaniel) [reply physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: So, is missing a documentation about the event's name, Lucas is working in a new document about possibles event's name in the LXCA and I'm waiting the link  for this document.

Comment (skovic) [reply physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: @CharlleDaniel Everything looks good. Is it OK if I go ahead and merge these changes? You can always create a PR later to update the link that depends on Lucas's document.

Comment (CharlleDaniel) [reply physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc]: @skovic no problem, go ahead. Thank you. :+1:

**Pull request merged**